packages/flutter_driver/lib/src/driver/vmservice_driver.dart

```diff
@@ -357,6 +357,11 @@ class VMServiceFlutterDriver extends FlutterDriver {
   Future<List<int>> screenshot() async {
     await Future<void>.delayed(const Duration(seconds: 2));
```

Comment:
Can we remove this now?

Reply:
Probably not as it still needs to wait some time for future frames to be rasterized before taking a screenshot.